src/analog/commands/uplink/getArchiveDaysMc.ts

```diff
@@ -53,9 +53,7 @@ export const id: types.TCommandId = 0x1b;
 export const name: types.TCommandName = 'getArchiveDaysMc';
 export const headerSize = 2;
 
-// date 2 bytes, channelList 1 byte (max channelList: 4), days 1 byte (max days - 255)
-// 4 + (4 channelList * 5 bytes of day values * 255 max days)
-const COMMAND_BODY_MAX_SIZE = 5104;
+const COMMAND_BODY_MAX_SIZE = 255;
```

Comment:
is it a hardcoded limit? where does it come from?

Reply:
The LoRaWAN protocol has a packet length limit of 255 bytes.
Also, our API does not support packets longer than 255 bytes, because we have a one-byte size field in the header.
The only thing is that the NB-IoT protocol supports a packet length of 1023 bytes, but there was no task to support a packet of this length.
So at the moment, our maximum packet length is 255.